src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -1,48 +1,209 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
 package org.carlmontrobotics.subsystems;
+import static org.carlmontrobotics.Constants.Arm.*;
 
-import org.carlmontrobotics.Constants.Arm.*;
+import org.carlmontrobotics.commands.ArmTeleop;
+import org.carlmontrobotics.lib199.MotorConfig;
+import org.carlmontrobotics.lib199.MotorControllerFactory;
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
 
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.util.sendable.SendableBuilder;
 import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.InstantCommand;
-import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj2.command.WaitCommand;
 
+// Arm angle is measured from horizontal on the intake side of the robot and bounded between -3π/2 and π/2
+// Wrist angle is measured relative to the arm with 0 being parallel to the arm and bounded between -π and π (Center of Mass of Roller)
 public class Arm extends SubsystemBase {
+  ///GOALS
+  /*
+   * public static final double intakeAngle = Math.toRadians(0);
+		public static final double ampAngle = Math.toRadians(103);
+		public static final double placeholderSpeakerAngle1 = Math.toRadians(24);
+		public static final double placeholderSpeakerAngle2 = Math.toRadians(24);
+		public static final double placeholderSpeakerAngle3 = Math.toRadians(24);
+		public static final double climberUpAngle = Math.toRadians(24);
+		public static final double climberDownAngle = Math.toRadians(24);
+   */
+    // a boolean meant to tell if the arm is in a forbidden posistion AKA FORBIDDEN FLAG
+    
+    private final CANSparkMax armMotor1 = MotorControllerFactory.createSparkMax(ARM_MOTOR_PORT_1, MotorConfig.NEO);
+    private final CANSparkMax armMotor2 = MotorControllerFactory.createSparkMax(ARM_MOTOR_PORT_2, MotorConfig.NEO);
```

Comment:
Rename arm motors to whichever one is master or follower. If you tell a follow motor to move it will not move the master motor. If you tell the master motor to move it will also move the follow motor.